internal/resourcemodifiers/resource_modifiers_test.go

```diff
@@ -9,6 +9,10 @@ import (
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 )
 
 func TestGetResourceModifiersFromConfig(t *testing.T) {
```

Comment:
can you add deserialization tests for new contracts?

Reply:
what is the perpose of the deserialization tests? we just call `yaml.UnmarshalStrict` to convert the yaml manifest to go struct.

Reply:
Just to ensure that there are no missed fields,  for example fields that are not annotated properly with `json` tags.  Or any missed nullable fields which might have been made compulsory by mistake or vica versa.
And all in all it can give a good payload for using in documentation.

Reply:
added